### DISCUSSION TITLE
Themes

### DISCUSSION THREAD

**Discussion by Cristalnoir** [Mezryss/FVTT-Genesys#154] [Ideas]
Having edited the CSS file to make the character sheet a little more medieval fantasy. With each update, it is difficult to reimplement my previous work. Will the Genesys system for Foundry VTT support themes for character sheets. This will avoid accidents.

**Comment by Assembling-Kings** [comment]
This is not something that is on the current roadmap to 1.0.0, and truthfully is not something that I'm considering to do in the near future. I might revisit this idea after 1.0.0.
